src/docfx/build/redirection/RedirectionProvider.cs

```diff
@@ -173,6 +173,18 @@ private static RedirectionItem[] LoadRedirectionModel(ErrorBuilder errors, strin
                     // Rebase source_path based on redirection definition file path
                     var basedir = Path.GetDirectoryName(fullPath) ?? "";
 
+                    // Give a warning when source-path not specified in .openpublishing.redirection.json
+                    if (fullPath.EndsWith(".openpublishing.redirection.json"))
```

Comment:
this warning should include `renames` as well

Reply:
Yes, no matter where the redirection rules come from, we should always have this validation.

---
In reply to: [466897413](https://github.com/dotnet/docfx/pull/6377#discussion_r466897413) [](ancestors = 466897413)

Reply:
Done